gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -16698,8 +16743,8 @@ properties:
     enum:
       - "yes"
       - "no"
-      - unknown
-      - not reported
+      - Unknown
+      - Not Reported
       - Not Allowed To Collect
```

Comment:
Will this require deprecation-migration?

Reply:
hmm not sure, good point. I'll revert the changes for this and other like it.